### DISCUSSION TITLE
Upgrading from 3.5.8 to 3.6-beta2

### DISCUSSION THREAD

**Discussion by haki99** [netbox-community/netbox#13512] [Help Wanted!] [answered]
Hello everyone!

I tried to upgrade our test netbox instance from 3.5.8 to 3.6-beta2 and got the following error after starting netbox:

(Debug mode is enabled for detailed debugging)

![image](https://github.com/netbox-community/netbox/assets/51157531/860be4e8-69be-4cd0-ac91-cb30efdf55a8)

We are using the following plugins, could these be the problem due to incompatibility or is it something else?

- django3_auth_saml2
- netbox-plugin-auth-saml2
- xlsxwriter
- requests
- smbprotocol
- paramiko
- django-auth-ldap

Also during the start the logs says the following:

```
Operations to perform:
  Apply all migrations: account, admin, auth, circuits, contenttypes, core, dcim, django_rq, extras, ipam, sessions, social_django, taggit, tenancy, users, virtualization, wireless
Running migrations:
  No migrations to apply.
  Your models in app(s): 'circuits', 'core', 'dcim', 'extras', 'ipam', 'tenancy', 'users', 'virtualization', 'wireless' have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

What I could figure out from previous issues/discussions is that this makemigrations is a **very** developer script and after trying it, it says that everything is up-to-date, nothing to migrate.

I tried to delete all of the social auth profiles but it doesn't solved the problem.

After hours of debugging I am unable to solve this issues so I'm asking for help.

**Comment by kkthxbye-code** [comment, marked as answer]
I created the following issue, which is what you ran into: https://github.com/netbox-community/netbox/issues/13513

As a workaround you can set LOGIN_REQUIRED = True if that's possible for your usecase.

**Comment by haki99** [comment]
Thank you, it solved my problem. I read your opened issue, I think it would be beneficial, especially for the seamless upgrade between the two sub versions.

Or at least mentioning this issue in the "upgrade documents".
